src/install/kafka/whatsNext.mdx

```diff
@@ -0,0 +1,4224 @@
+    }
+
+    <tr>
+      <td>
+        **TOPIC_LIST**
+      </td>
+
+      <td>
+        JSON array of topic names to monitor. Only in effect if `topic_mode` is set
+        to `list`.
+      </td>
+
+      <td>
+        \[]
+      </td>
+
+      <td style={{ 'text-align': 'center' }}>
+        M/I
+      </td>
+    </tr>
+
+    {
+      ' '
+    }
+
+    <tr>
+      <td>
+        **TOPIC_REGEX**
+      </td>
+
+      <td>
+        Regex pattern that matches the topic names to monitor. Only in effect if
+        `topic_mode` is set to `regex`.
+      </td>
+
+      <td>
+        N/A
+      </td>
+
+      <td style={{ 'text-align': 'center' }}>
+        M/I
+      </td>
+    </tr>
+
+    <tr>
+      <td>
+        **TOPIC_BUCKET**
+      </td>
+
+      <td>
+        Used to split topic collection across multiple instances. Should be of the form `<bucket number>/<number of buckets>`.
+      </td>
+
+      <td>
+        1/1
+      </td>
+
+      <td style={{ "text-align": "center" }}>
+        M/I
+      </td>
+    </tr>
+
+    <tr>
+      <td>
+        **COLLECT_TOPIC_SIZE**
+      </td>
+
+      <td>
+        Collect the metric Topic size. Options are `true` or `false`, defaults to `false`.
+
+        **This is a resource-intensive metric to collect, especially against many topics.**
+      </td>
+
+      <td>
+        false
+      </td>
+
+      <td style={{ "text-align": "center" }}>
+        M/I
+      </td>
+    </tr>
+
+    <tr>
+      <td>
+        **COLLECT_TOPIC_OFFSET**
+      </td>
+
+      <td>
+        Collect the metric Topic offset. Options are `true` or `false`, defaults to `false`.
+
+        **This is a resource-intensive metric to collect, especially against many topics.**
+      </td>
+
+      <td>
+        false
+      </td>
+
+      <td style={{ "text-align": "center" }}>
+        M/I
+      </td>
+    </tr>
+  </tbody>
+</table>
+
+## Configure KafkaConsumerSample and KafkaProducerSample collection [#KafkaConsumerSample-collection]
+
+The Kafka integration collects both Metrics(<strong>M</strong>) and Inventory(<strong>I</strong>) information. Check the **Applies To** column below to find which settings can be used for each specific collection:
+
+<table>
+  <thead>
+    <tr>
+      <th style={{ width: "150px" }}>
+        Setting
+      </th>
+
+      <th>
+        Description
+      </th>
+
+      <th>
+        Default
+      </th>
+
+      <th>
+        Applies To
+      </th>
+    </tr>
+  </thead>
+
+  <tbody>
+    <tr>
+      <td>
+        **CLUSTER_NAME**
+      </td>
+
+      <td>
+        User-defined name to uniquely identify the cluster being monitored. **Required**.
+      </td>
+
+      <td>
+        N/A
+      </td>
+
+      <td style={{ "text-align": "center" }}>
+        M/I
+      </td>
+    </tr>
+
+    <tr>
+      <td>
+        **PRODUCERS**
+      </td>
+
+      <td>
+        Producers to collect. For each producer a `name`, `hostname`, `port`, `username`, and `password` can be specified in JSON form.
```

Comment:
You can specify X Y Z for each producer using JSON form

or something like that to fix the passive